ide/defaults/src/org/netbeans/modules/defaults/BlueTheme-editor.xml

```diff
@@ -62,4 +62,9 @@
     <fontcolor name="LastOperation" bgColor="6f9f5a"/>
     <fontcolor name="StepOutOperation" bgColor="2D6035"/>
 
+    <fontcolor name="nbcoverage-covered" bgColor="0F4F40"/>
+    <fontcolor name="nbcoverage-uncovered" bgColor="4F0F40"/>
+    <fontcolor name="nbcoverage-inferred" bgColor="1F4F40"/>
+    <fontcolor name="nbcoverage-partial" bgColor="909040"/>
```

Comment:
Why "nbcoverage"? Perhaps just "coverage"? (Everything in NetBeans is NetBeans...)